### ISSUE TITLE
Add "Disk -> Save" menu item

### ISSUE THREAD

**Issue by feraligatr** [dirkwhoffmann/virtualc64#267]
Sorry, I don't know where else I can get help.
I tried to make a new adventure disk (F1) in ACS (Adventure Construction Set). First it asked me to insert my new adventure disk. I dragged a newly created D64 file into VirtualC64 window (I've tried both LOAD "*",8,1 and Flash into memory). After some time writing, it asked me to change to ACS disk. Then I dragged ACS Disk into it. After some time reading, it asked me again to put in my new disk. Then I dragged my new D64 into it again. This time ACS failed to continue. I guess the data was not written correctly in the previous step. I tried VICE and it worked correctly.
Is it a bug of VirtualC64, or something I didn't do right?
BTW, is there a way to create new disk images directly in VirtualC64?

**Comment by dirkwhoffmann** [comment]
It seems to be a bug. I was able to create an adventure disk, but after creation, ACS does not accept it as such. I fear that the drive implementation needs to be improved before this is working.

**Comment by Alessandro1970** [comment]
Hi to all,

with VirtualC64 2.1 it works correctly but when you have to create the game disk you need to stick to the following:
every time **before changing the disk**, it is necessary to **save the progress** with the **SAVE AS** command in the **DISK menu** of the VirtualC64.

Probably it would have been quicker to have a SAVE command and not SAVE AS because it does not overwrite but creates (exports) the .d64 to another location (Documents) and so it is necessary to use this file every time until the creation procedure is finished.
**However it works for me.**
<img width="762" alt="schermata 2018-06-30 alle 09 07 28" src="https://user-images.githubusercontent.com/34916641/42122681-00b9d082-7c46-11e8-9c64-e1164ad348dc.png">

<img width="761" alt="schermata 2018-06-30 alle 09 06 42" src="https://user-images.githubusercontent.com/34916641/42122682-0490c27e-7c46-11e8-8312-3ffe48372f49.png">

**Comment by Ottavius** [comment]
ok, I managed to do it completely
But everything is very convoluted, I'll try again when the SAVE command will be available to save directly on the disk in use.

**Comment by dirkwhoffmann** [comment]
"Export Recent" has been added to the "Disk" menu:
http://www.dirkwhoffmann.de/virtualC64/VirtualC64_2.3_beta3.zip

**Comment by dirkwhoffmann** [comment]
Please don't test this feature. I am already working on this again.

**Comment by dirkwhoffmann** [comment]
I applied some more changes in the fourth beta:
http://www.dirkwhoffmann.de/virtualC64/VirtualC64_2.3_beta4.zip

1. Only the last used disk URL is remembered and shown in the "Export Disk To" menu.
   In beta3, I had a list of 10 items as in InsertRecent, but I found this confusing. Usually, one wants to export quickly to the currently used file and having a list makes it more likely that a file get overwritten by mistake. 

2. The save status is now reflected in the icon at the bottom bar. When a disk is inserted, the icon is deep black. When something is written to disk, it gets gray to indicate that it has unsaved data on it. 

3. I changed the way, disks are ejected and inserted and fixed a bug related to blocking and unblocking the light barrier. If I remember correctly, some user had reported a problem with unrecognized disk changes. Hopefully, these issues are fixed now. 

4. The Cartridge menu has been renamed to "Expansion Port" and got a "Attach Recent" menu similar to the "Insert Recent" disk menu. 

5. The "Disk" menu has been renamed to "Drive", because once we have two drives, we need two menus "Drive 1" and "Drive 2" which sounds more reasonable to me than "Disk 1" and "Disk 2". 

To get this done, I had to change a lot of stuff internally, so I hope I didn't break too much.
